### DISCUSSION TITLE
how to check the version code of termux with command

### DISCUSSION THREAD

**Discussion by Exanoff12** [termux/termux-packages#9544] [Q&A] [answered]
Hi,
so I (and some other people) are creating a sh script for installing windows 7 on android via the use of termux. and we want so that the script will only work if the script was executed in app version code higher than 117. we can easily do this by using "if statement" but to do this we need to find a way to get the version code of termux. so is there any way we can check the version code using commands ? if possible please let us know.

thanks in advance
(probably the worst formatted question. sorry)

**Comment by licy183** [comment, marked as answer]
https://github.com/termux/termux-packages/blob/983a8f62a86be167ec8fbe7f8aa3442489167004/packages/termux-tools/termux-info#L75-L78
Maybe you can get the application version from the environment variable `$TERMUX_VERSION`.

**Comment by agnostic-apollo** [comment]
`dpkg --compare-versions "$TERMUX_VERSION" ge 0.118.0`

**Reply by Exanoff12** [reply]
thanks

**Comment by agnostic-apollo** [comment]
`dpkg --compare-versions "$TERMUX_VERSION" ge 0.118.0`

**Reply by Exanoff12** [reply]
thanks
